sdk/python3/README.md

```diff
@@ -1,19 +1,174 @@
+# Confidential Cloud Native Primitives client library for Python
```

Comment:
Confidential Cloud-Native Primitives client library for Python

BTW: a generation comment: Cloud-Native is used as in the name of CCNP, so please search/replace it in CCNP context.